package.json

```diff
@@ -2,38 +2,48 @@
   "private": true,
   "name": "tlon-apps",
   "type": "module",
-  "workspaces": [
-    "apps/*",
-    "packages/*"
-  ],
   "scripts": {
-    "build:shared": "npm run build --prefix ./packages/shared",
-    "build:ui": "npm run build --prefix ./packages/ui",
-    "build:editor": "npm run build --prefix ./packages/editor",
-    "build:web": "npm run build --prefix ./apps/tlon-web",
-    "build:mobile": "npm run build --prefix ./apps/tlon-mobile",
-    "build:apps": "npm run build:mobile",
-    "build:packages": "npm run build:shared && npm run build:ui && npm run build:editor",
-    "build:all": "npm run build:packages && npm run build:apps",
-    "dev:shared": "npm run dev --prefix ./packages/shared",
-    "dev:android": "concurrently \"npm run dev:shared\" \"npm run dev --prefix ./apps/tlon-mobile\" \"npm run android --prefix ./apps/tlon-mobile\"",
-    "dev:ios": "concurrently \"npm run dev:shared\" \"npm run dev --prefix ./apps/tlon-mobile\" \"npm run ios --prefix ./apps/tlon-mobile\" \"npm run watch --prefix ./packages/ui\"",
-    "dev:web": "concurrently \"npm run dev:shared\" \"npm run dev-no-ssl --prefix ./apps/tlon-web\"",
-    "test": "npm run test --workspaces --if-present -- run -u",
+    "build:shared": "pnpm --filter '@tloncorp/shared' build",
+    "build:ui": "pnpm --filter '@tloncorp/ui' build",
+    "build:editor": "pnpm --filter '@tloncorp/editor' build",
+    "build:web": "pnpm --filter 'tlon-web' build",
+    "build:mobile": "pnpm --filter 'tlon-mobile' build",
+    "build:apps": "pnpm run build:mobile",
+    "build:packages": "pnpm run build:shared && pnpm run build:ui && pnpm run build:editor",
+    "build:all": "pnpm run build:packages && pnpm run build:apps",
+    "dev:shared": "pnpm --filter '@tloncorp/shared' dev",
+    "dev:android": "concurrently \"pnpm run dev:shared\" \"pnpm --filter 'tlon-mobile' dev\" \"pnpm --filter 'tlon-mobile' android\"",
+    "dev:ios": "concurrently \"pnpm run dev:shared\" \"pnpm run dev --filter 'tlon-mobile'\" \"pnpm --filter 'tlon-mobile' ios\" \"pnpm --filter '@tloncorp/ui' watch\"",
+    "dev:web": "concurrently \"pnpm run dev:shared\" \"pnpm --filter 'tlon-web' dev-no-ssl\"",
+    "test": "pnpm run -r test run -u",
     "prepare": "husky",
-    "postinstall": "npx patch-package && npm run build:all"
+    "postinstall": "pnpm run build:all"
   },
   "devDependencies": {
     "@trivago/prettier-plugin-sort-imports": "^4.3.0",
     "concurrently": "^8.2.2",
     "husky": "^9.0.10",
-    "patch-package": "^6.4.7",
+    "lint-staged": "^15.0.0",
```

Comment:
do these do the same thing? I think we still use patch package

Reply:
They don't do the same thing -- this is just moving the lint-staged command to top level. PNPM has patching built in, so the patches in `<root>/patches` get applied automatically on install. More info here: https://pnpm.io/cli/patch